### ISSUE TITLE
distribute library wheel on pypi

### ISSUE THREAD

**Issue by proppy** [googlesamples/assistant-sdk-python#49]
now that https://github.com/pypa/warehouse/issues/2003 is fixed, we should distribute the library wheel on pypi.

**Comment by proppy** [comment]
https://pypi.python.org/pypi/google-assistant-library

**Comment by bennuttall** [comment]
Am I right in thinking that next-gen PyPI now allows uploading arm wheels, and they appear on legacy PyPI?

If so, how do you do this?

**Comment by proppy** [comment]
@bennuttall sorry for the late reply, I didn't see your comment after the issue was closed.

The relevant commit for warehouse are:
https://github.com/pypa/warehouse/pull/2010
https://github.com/pypa/warehouse/pull/2012

And looking at the code:
https://github.com/pypa/warehouse/blob/de28c169cc683d0e60bf78d6b9485a19caf5e869/warehouse/forklift/legacy.py#L574
https://github.com/pypa/warehouse/blob/54e59eab93be8aee21d96d993b64bf53272b9087/warehouse/legacy/api/pypi.py#L29

I could find the new codepath for `upload.pypi.org`, so I'd assume that the `forklift.legacy` path is still used.

**Comment by bennuttall** [comment]
Yeah I saw - great stuff!

I completed a build run of all 750k package releases on PyPI, with a 76% success rate. I'm currently working on another build with some key dependencies installed, and I expect to see an 85% success rate.

See the project at https://www.piwheels.hostedpi.com/, on github at https://github.com/bennuttall/piwheels and read all about it at http://bennuttall.com/piwheels-building-a-faster-python-package-repository-for-raspberry-pi-users/

We skip packages with no source distribution, so we won't host `google-assistant-library`, but since a wheel is available from PyPI, people can get it from there the usual way.

For something like `tensorflow`, which does not provide a source distribution, we may grab a wheel built elsewhere, i.e. https://github.com/samjabrahams/tensorflow-on-raspberry-pi and add manually.
